replit/nixmodules
# better extensions for bun

Why
===

i didn't do a complete list of extensions for bun when i made the module 😬

What changed
============

better extensions definitions for bun

Test plan
=========

in a bun repl:
- `mv index.ts index.mts`
- change `entrypoint` in `.replit` to `index.mts`
- run button works

Rollout
=======

_Describe any procedures or requirements needed to roll this out safely (or check the box below)_
